### PR TITLE
Add integration with circleci

### PR DESCRIPTION
**Pull request by MathiasLantean**
A code refactoring was made during this process as it was also added flake8 to the project.

### CODE REVIEW

**Review by kaozdl — APPROVED** [review]
Consider adding factory boy for future tests

File: base/settings/develop.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = ['*']
 
-INSTALLED_APPS += [
-]
+INSTALLED_APPS += [  # noqa
+]  # noqa
```
Comment (guilleijo) [inline base/settings/develop.py:10]: Is this setting necessary if you are not adding something?

**Review by guilleijo — APPROVED** [review]
Left a comment but nothing serious

**Comment by guilleijo** [comment]
> Consider adding factory boy for future tests

+1 to this

File: base/settings/develop.py (hunk 1 above)
Comment (MathiasLantean) [reply base/settings/develop.py:10]: Good catch! As I added in the explanation of the commit comment.  Add '# noqa' tag for specific lines which make flake8 fail but they actually are correct.
Flake8 suggests that INSTALLED_APPS could be not defined but as I imported the base settings it always will be defined.

**Pull request merged**